### ISSUE TITLE
available hardware

### ISSUE THREAD

**Issue by pinkit-cwa** [Lurkars/esp-ena#6]
@Lurkars yesterday I stumbled upon a company/brand called [LILYGO](http://www.lilygo.cn/)  . They seem to have a lot of ESP32 based affordable hardware, including display and RTC, and provide that for DIY projects.

I bought [this](http://www.lilygo.cn/claprod_view.aspx?TypeId=21&Id=1315&FId=t28:21:28) on aliexpress for about 18 EUR, for evaluation purposes:
- ESP32, WIFI, Bluetooth
- 4MB flash

**Comment by einsiedlerkrebs** [comment]
[M5StickC](https://m5stack.com/products/stick-c) (like M5stack) a little ESP32 device in small format with case and battery would be a good device to start with as well. I will try it out and let you know.

**Comment by Lurkars** [comment]
Yes, thanks. I am curently working with a team on a M5StickC (Plus) version and it will get published soon!

**Comment by einsiedlerkrebs** [comment]
Yeah. Looking forward to it. Thanks!!1!

**Comment by einsiedlerkrebs** [comment]
Hey there, @Lurkars is there any progress/ can I check something out for the M5StickC+ variant?

**Comment by Lurkars** [comment]
Sorry to inform, that we have some problems with display drivers. I will checkout, when we make the proxy server public, that at least the ENA side is done.

**Comment by Lurkars** [comment]
As I discovered https://github.com/nopnop2002/esp-idf-m5stickC-Plus which was created only 12 days ago, I now have a functional display driver and will start porting my display+interface code to m5stickc+. Hope that won't take too long. Problem is the input, because I currently use 7 buttons and stick only have 2, but I will test how usable input over acc/gyro will be with tilting/shaking or an simple input extension over I2C.

**Comment by Lurkars** [comment]
M5Stick (PLUS) support added, need to choose device in menuconfig ENA_INTERFACE -> ENA_INTERFACE_DEVICE
and (un)comment line in components/interface/CmakeLists.txt before building and flashing.
